vernac/classes.ml

```diff
@@ -28,8 +28,8 @@ module RelDecl = Context.Rel.Declaration
 module NamedDecl = Context.Named.Declaration
 (*i*)
 
-let set_typeclass_transparency c local b =
-  let locality = if local then Goptions.OptLocal else Goptions.OptGlobal in
+let set_typeclass_transparency ?(locality=Goptions.OptLocal) c b =
```

Comment:
```suggestion
let set_typeclass_transparency ~locality c b =
```
no need for this to be implicit

Reply:
Why not?

Reply:
Implicit arguments make it harder to tell what's going on so should be avoided unless there's a benefit